### PR TITLE
Update to NET8

### PR DESCRIPTION
**Pull request by daniel-almeida-konkconsulting**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [X] I ensured that the PR title is good enough for the changelog.
-   [X] I labeled the PR.

### CODE REVIEW

File: AdminUi/src/AdminUi.Infrastructure/Persistence/IServiceCollectionExtension.cs [hunk 1]
```diff
@@ -48,6 +49,7 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, A
                             sqlOptions.CommandTimeout(20);
                             sqlOptions.MigrationsAssembly(POSTGRES_MIGRATIONS_ASSEMBLY);
                             sqlOptions.EnableRetryOnFailure(options.RetryOptions.MaxRetryCount, TimeSpan.FromSeconds(options.RetryOptions.MaxRetryDelayInSeconds), null);
+                            sqlOptions.MigrationsHistoryTable(HistoryRepository.DefaultTableName, "AdminUi");
```
Comment (tnotheis) [inline AdminUi/src/AdminUi.Infrastructure/Persistence/IServiceCollectionExtension.cs:52]: Can you please add a Todo comment here (and at every other place at which we set the migrations table) that remembers us to remove this as soon as the bug is fixed? (add a link to the bug)

**Review by tnotheis — CHANGES_REQUESTED** [review]
Can you please try to upgrade all nuget packages again? Just so we can be sure to have the latest versions, including the once that were published after your initial upgrade.
And you should unexclude the npgsql package in dependabot.yml.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,21 +1,5 @@
...
-        patterns:
-          - "*"
-        exclude-patterns:
-          - "Npgsql.EntityFrameworkCore.PostgreSQL"
-    reviewers:
-      - "tnotheis"
-    labels:
-      - "dependencies"
-
   ##### GitHub Actions #####
```
Comment (tnotheis) [inline .github/dependabot.yml:3]: What are you doing??? :D :D
I was just talking about the removal of the exclude of the Npgsql package. Not about disabling the whole configuration for NuGet

**Review by tnotheis — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (daniel-almeida-konkconsulting) [reply .github/dependabot.yml]: You saw nothing 🤫

**Review by tnotheis — DISMISSED** [review]

**Review by tnotheis — DISMISSED** [review]

**Review by tnotheis — APPROVED** [review]

**Pull request merged**